cpp/bench/ann/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ option(CUVS_ANN_BENCH_USE_CUVS_BRUTE_FORCE "Include cuVS brute force knn in benc
 option(CUVS_ANN_BENCH_USE_CUVS_CAGRA_HNSWLIB "Include cuVS CAGRA with HNSW search in benchmark" ON)
 option(CUVS_ANN_BENCH_USE_HNSWLIB "Include hnsw algorithm in benchmark" ON)
 option(CUVS_ANN_BENCH_USE_GGNN "Include ggnn algorithm in benchmark" OFF)
+option(CUVS_ANN_BENCH_USE_CUVS_MG "Include cuVS ann mg algorithm in benchmark" ON)
```

Comment:
What's going to happen if we build with this argument ON but BUILD_MG_ALGOS is OFF? I think we should just default this to whatever $BUILD_MG_ALGOS is set to. That makes it easier and puts the control in a single place.

Reply:
Agree, we get compilation error if the flags do not align.